src/middleware/middleware.ts

```diff
@@ -11,11 +12,11 @@ import { useNavStore } from '~/stores/nav'
  * add `?embedded=false` to the end of the URL.
  *
  * Messages sent to the outer window have the following format:
- * {type: <event type>, value: <event value>}.
+ * `{type: <event type>, value: <event value>}`.
  * Currently, one event type is used:
  * - `urlChange` sends the relative path of the URL on every URL change.
  */
-export default function ({ query, route, $pinia }) {
+const middleware: Middleware = ({ query, route, $pinia }: Context) => {
```

Comment:
nit: The `Context` annotation is redundant now

```suggestion
const middleware: Middleware = ({ query, route, $pinia }) => {
```